src/Features/Core/Portable/Formatting/IEditorFormattingService.cs

```diff
@@ -8,7 +8,7 @@
 using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Text;
 
-namespace Microsoft.CodeAnalysis.Editor
+namespace Microsoft.CodeAnalysis
```

Comment:
i don't think we want to place this root namespace.  i'm also worried about calling it IEditor in a layer tht is unrelated to editors.  @jasonmalinowski